src/views/Account.vue

```diff
@@ -24,7 +24,7 @@
             <span
               class="account-container_balance_value"
               :id="`${asset}_balance_value`"
-              :style="{ fontSize: formatFontSize(balance) }"
+              :style="{ fontSize: '30px' }"
```

Comment:
Is there a reason this change isn't done in the css class?

Reply:
This has been updated, @monokh.